app/javascript/dashboard/components-next/button/Button.vue

```diff
@@ -184,7 +184,7 @@ const STYLE_CONFIG = {
     center: 'justify-center',
     end: 'justify-end',
   },
-  base: 'inline-flex items-center min-w-0 gap-2 transition-all duration-200 ease-in-out border-0 rounded-lg outline-1 outline disabled:opacity-50',
+  base: 'inline-flex items-center min-w-0 gap-2 transition-all duration-200 ease-in-out border-0 rounded-lg outline-1 outline disabled:opacity-50 outline-offset-[-1px]',
```

Comment:
Why the need to change the outline offset?

Reply:
@scmmishra The reason I added it because of this, 
**After**
<img width="744" alt="image" src="https://github.com/user-attachments/assets/91854bc9-d0e5-4be6-b1f0-a40a27ef9cc6" />

**Before**
<img width="744" alt="image" src="https://github.com/user-attachments/assets/15807447-b570-4876-ba20-c6c8aff3687e" />

If it's not needed then I can remove it.